sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -84,46 +92,85 @@ async function main() {
     throw appComponentCreationResult.body.error;
   }
 
-  // Creating the test run
-  const testRunCreationResult = await client.path("/testruns/{testRunId}", testRunId).patch({
+  // Creating/Updating the test run
+  const testRunPoller = await beginCreateOrUpdateTestRun(client, testRunId, {
     contentType: "application/merge-patch+json",
     body: {
-      testId: testCreationResult.body.testId,
+      testId: testId,
       displayName: displayName,
-      vusers: 10,
+      virtualUsers: 10,
     },
   });
+  const testRunResult = await testRunPoller.pollUntilDone({
+    abortSignal: AbortController.timeout(60000), // timeout of 60 seconds
+  });
 
-  if (isUnexpected(testRunCreationResult)) {
-    throw testRunCreationResult.body.error;
-  }
+  if (testRunPoller.getOperationState().status != "succeeded" && testRunResult)
+    throw new Error("There is some issue in running the test, Error Response : " + testRunResult);
```

Comment:
Could we throw the `testRunPoller.getOperationState().error` instead?

Reply:
Done.